### ISSUE TITLE
RHEL 8/9 support

### ISSUE THREAD

**Issue by jcpunk** [jhoblitt/puppet-mcelog#13]
Can this be updated for RHEL8 and RHEL9?

**Comment by jhoblitt** [comment]
Yes... It needs a plumbing / CI update and then it looks like modernization...

**Comment by jhoblitt** [comment]
@jcpunk If I get github actions going are you willing to volunteer to get the accept tests working on el8/el9?

**Comment by jcpunk** [comment]
If you can get it on PDK.  My local runs of non-pdk based puppet tests are depressingly hit or miss.

**Comment by jhoblitt** [comment]
I'm not sure what you mean by "on PDK"... Do you mean the default `pdf` template created `Gemfile`, `Rakefile`, etc.?  I generally try to stay in sync with voxpupuli.

**Comment by jcpunk** [comment]
I've had a lot of luck with https://puppet.com/docs/pdk/2.x/pdk.html

**Comment by jcpunk** [comment]
If you're open to it, I can try and work up the migration path.  This module doesn't look too complex.

**Comment by jhoblitt** [comment]
Doh, I already started on the update... should push a branch soonish.

**Comment by jhoblitt** [comment]
@jcpunk One question I have is if the `$service_stdout` param (don't like the name either...) is needed and if there is any reason to overwrite the systemd service unit?

**Comment by jhoblitt** [comment]
I'm thinking that the `config_file_template` param should go as well and the caller can pass just a string.

**Comment by jhoblitt** [comment]
There is no automagic beaker target from rhel 9 (or any rhel, or centos stream)... I've added rhel9 to the `metadata.json` in #14 but I don't know if it actually works.

**Comment by jhoblitt** [comment]
@jcpunk Could you give #14 a review?
